postgres/lib.w

```diff
@@ -45,16 +42,52 @@ pub class Database {
   }
 }
 
-pub class DatabaseNeon impl IDatabase {
+class DatabaseSim impl IDatabase {
+  url: str?;
+  new(props: DatabaseProps) {
+    let image = "postgres:${props.pgVersion ?? 15}";
+    let container = new containers.Workload(
+      name: "postgres",
+      image: image,
+      env: {
+        POSTGRES_PASSWORD: "password"
+      },
+      port: 5432,
+      public: true,
+    );
+    this.url = container.getPublicUrl();
```

Comment:
```suggestion
    this.url = container.publicUrl;
```

Reply:
I was curious what's the difference between them

Reply:
It's because Interfaces don't support fields and I wanted to enforce the api.

Pretty ugly.